File: programme/fibonaci-series-without-using-recursion/fibonaci.c
// Write a program to print fibonacci series without using recursion
#include <stdio.h>

int main()
{
    int a = 0, b = 1, temp = 0, n;
    printf("Input  : ");
    scanf("%d", &n);
    int i = 0;
    printf("Output : ");
    while (i != n)
    {
        printf("%d ", a);
        temp = a;
        a = b;
        b = temp + b;
        i++;
    }
    printf("\n");
    return 0;
}

File: programme/fibonaci-series-without-using-recursion/fibonaci.js
// Write a programmeto print Fibonacci series without recursion

"use strict";

// perform following command to run programme
// node fibonaci.js

const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

rl.question("Input  : ", (n) => {
  n = Number(n);
  let [a, b] = [0, 1];
  if (isNaN(n)) {
    console.log("Some error occured");
  } else {
    rl.write("Output :");
    for (let i = 0; i < n; i++) {
      rl.write(` ${a}`);
      let tmp = a;
      a = b;
      b = tmp + b;
    }
    console.log();
  }
  rl.close();
});
